assets/app/features/upgrade/controllers/upgrade-landing.controller.spec.js

```diff
@@ -3,6 +3,135 @@
   crowbarFactory $q $rootScope module $state */
 describe('Upgrade Landing Controller', function() {
     var controller,
+        completedUpgradeResponseData = {
+            current_step: 'upgrade_prepare',
```

Comment:
current_step should be `admin_backup` here, as `upgrade_prepare` is already passed